src/babashka/fs.cljc

```diff
@@ -1256,9 +1256,9 @@
          {:keys [charset]
           :or {charset "utf-8"}} opts
          opts [:encoding charset]
-         old-val (apply slurp file opts)
+         old-val (apply slurp (as-file file) opts)
```

Comment:
should we rename the argument to `path`? even the docstring uses the word `path`.

Reply:
sure